src/main/java/de/codingair/clansystem/utils/clan/Clan.java

```diff
@@ -128,15 +326,14 @@ public UUID getMemberByRank(Rank rank) {
         } else return null;
     }
 
-    public boolean hasPermission(UUID uuid, Permission permission) {
-        Rank r = getRank(uuid);
-        return r != null && r.hasPermission(permission);
-    }
-
     public Rank getRank(UUID uuid) {
         return members.get(uuid);
     }
 
+    public Rank getRank(int id) {
```

Comment:
To avoid conflicts with the other `getRank()`, I would like this method name
```suggestion
    public Rank getClanRank(int id) {
```